package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "mali-error-mapper",
+  "version": "0.0.1",
+  "description": "Error mapper for Mali applications",
+  "keywords": [
+    "error",
+    "grpc",
+    "mali",
+    "mapper"
+  ],
+  "homepage": "https://github.com/uphold/mali-error-mapper#readme",
+  "bugs": {
+    "url": "https://github.com/uphold/mali-error-mapper/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/uphold/mali-error-mapper.git"
+  },
+  "license": "MIT",
+  "author": "Uphold",
+  "main": "src/index.js",
+  "scripts": {
+    "changelog": "uphold-scripts changelog $npm_package_version",
+    "lint": "uphold-scripts lint",
+    "release": "uphold-scripts release",
+    "test": "uphold-scripts test",
+    "version": "uphold-scripts version"
+  },
+  "pre-commit": [
+    "lint"
+  ],
+  "dependencies": {
+    "lodash": "^4.17.15",
+    "sf": "^0.2.0"
+  },
+  "devDependencies": {
+    "@uphold/http-errors": "git+ssh://git@github.com/uphold/http-errors#v1.2.0",
+    "@uphold/validator.js": "git+ssh://git@github.com/uphold/uphold-validator.js#v0.1.0",
```

Comment:
`uphold/http-errors` and `uphold/uphold-validator.js` are both private repos. So we either make them open source or we'll have to make this one private as well. My preference would, of course, to go with the former, especially since those are trivial wrappers around other libraries.

Reply:
We went with turning the repos public available.